geode-core/src/test/java/org/apache/geode/internal/cache/PartitionedRegionTest.java

```diff
@@ -61,7 +76,134 @@ public void setup() {
         new PartitionAttributesFactory().setTotalNumBuckets(1).setRedundantCopies(1).create());
     partitionedRegion = new PartitionedRegion(regionName, attributesFactory.create(),
         null, internalCache, mock(InternalRegionArguments.class));
+    DistributedSystem mockDistributedSystem = mock(DistributedSystem.class);
+    when(internalCache.getDistributedSystem()).thenReturn(mockDistributedSystem);
+    when(mockDistributedSystem.getProperties()).thenReturn(gemfireProperties);
+  }
+
+  @Test
+  @Parameters(method = "parametersToTestUpdatePRNodeInformation")
+  public void verifyPRConfigUpdatedAfterLoaderUpdate(CacheLoader mockLoader, CacheWriter mockWriter,
+      byte configByte) {
+    gemfireProperties.put(ENABLE_CLUSTER_CONFIGURATION, "true");
+    LocalRegion prRoot = mock(LocalRegion.class);
+    PartitionRegionConfig mockConfig = mock(PartitionRegionConfig.class);
+    PartitionedRegion prSpy = spy(partitionedRegion);
+
+    doReturn(prRoot).when(prSpy).getPRRoot();
+    when(prRoot.get(prSpy.getRegionIdentifier())).thenReturn(mockConfig);
+
+    InternalDistributedMember ourMember = prSpy.getDistributionManager().getId();
+    InternalDistributedMember otherMember = mock(InternalDistributedMember.class);
+    Node ourNode = mock(Node.class);
+    Node otherNode1 = mock(Node.class);
+    Node otherNode2 = mock(Node.class);
+    when(ourNode.getMemberId()).thenReturn(ourMember);
+    when(otherNode1.getMemberId()).thenReturn(otherMember);
+    when(otherNode2.getMemberId()).thenReturn(otherMember);
```

Comment:
Two `Node` with the same `MemberId`?

Reply:
Changed to use 2 mocked InternalDistributedMember instances